.circleci/config.yml

```diff
@@ -132,26 +124,24 @@ jobs:
   # Running with `use_multicore_sdr=true` should be integrated directly into the test code. For now we
   # just re-run the lifecycle tests to exercise the use_multicore_sdr code path with that setting set.
   test_multicore_sdr:
-    docker:
-      - image: filecoin/rust:latest
-    working_directory: /mnt/crate
-    resource_class: 2xlarge+
+    executor: default
+    environment: *setup-env
     steps:
-      - configure_environment_variables
       - checkout
       - attach_workspace:
           at: "."
       - restore_cache:
           keys:
-            - cargo-v28-{{ checksum "rust-toolchain" }}-{{ checksum "Cargo.toml" }}-{{ checksum "Cargo.lock" }}-{{ arch }}
+            - cargo-v28-b-{{ checksum "rust-toolchain" }}-{{ checksum "Cargo.toml" }}-{{ checksum "Cargo.lock" }}-{{ arch }}
       - restore_parameter_cache
       - run:
           name: Test with use_multicore_sdr pairing enabled
           command: |
             ulimit -n 20000
             ulimit -u 20000
             ulimit -n 20000
-            cargo +nightly -Zpackage-features test --all --verbose --release --test api -- --ignored lifecycle
+            cargo +nightly-2020-11-09 -Zpackage-features test --all --verbose --release lifecycle -- --ignored --nocapture
```

Comment:
Can be a followup PR: making the exactly nightly version (`nightly-2020-11-09`) a variable would be great. This would make updating to never versions easier (I already see myself wanting a newer nightly and then forgot to change it at all places).

Reply:
if you tell me how I can use a variable in the config, I would love to use one

Reply:
https://github.com/filecoin-project/bellperson/blob/929337fc19be417fc16d1aa3d3139d1e21fb6bdb/.circleci/config.yml#L3-L6 :)